DHFPROD-8275: Clean up Ant components: Dropdown, Menu and others

lucianaabc-ml opened the pull request:
### Description

#### Checklist: 
```diff
- Note: do not change the below
```

-  ##### Owner:

- [x] JIRA_ID included in all the commit messages
- [x] PR title is in the format JIRA_ID:Title
- [x] Rebase the branch with upstream
- [x] Squashed all commits into a single commit
- [x] Code passes ESLint tests
- [x] Added Tests
- [x] Run UI tests
  

- ##### Reviewer:

- [x] Reviewed Tests

Sanjeevani19 commented on marklogic-data-hub-central/ui/src/components/entities/mapping/entity-settings/entity-settings.tsx:
```diff
@@ -207,10 +206,10 @@ const EntitySettings: React.FC<Props> = (props) => {
                         id="targetPermissions"
                         placeholder="Please enter target permissions"
-                        data-testid={`${props.entityTitle}-targetPermissions`}
+                        dataTestid={`${props.entityTitle}-targetPermissions`}
```
@lucianaabc-ml I see data-testid has been replaced with dataTestid, that might be the reason for a couple of RTL test failures. It should be **data-testid**

Sanjeevani19 reviewed (commented):
@lucianaabc-ml there are couple of e2e failures on this PR https://jenkins.marklogic.com/blue/organizations/jenkins/Datahub_CI/detail/PR-6400/14/tests, Could you please look into them

Sanjeevani19 commented on marklogic-data-hub-central/ui/e2e/cypress/integration/modeling/graphValidations.spec.tsx:
```diff
@@ -93,7 +93,7 @@ describe("Graph Validations", () => {
   });
 
-  it("can center on entity type in graph view", {defaultCommandTimeout: 120000}, () => {
+  it.skip("can center on entity type in graph view", {defaultCommandTimeout: 120000}, () => {
```
@lucianaabc-ml any reason for skipping this test ?

lucianaabc-ml replied:
Yes, because with the graph dropdown commented, this functionality is not available so this test was breaking.

Sanjeevani19 replied:
@lucianaabc-ml Please add a comment that it should be handled as a part of DHFPROD-8284

Sanjeevani19 commented on marklogic-data-hub-central/ui/src/components/explore/graph-vis-explore/graph-vis-explore.tsx:
```diff
@@ -301,16 +300,16 @@ const GraphVisExplore: React.FC<Props> = (props) => {
   };
 
-  const menuClick = async (event) => {
+  /*   const menuClick = async (event) => {
```
@lucianaabc-ml Let's add a comment here mentioning the ticket number DHFPROD-8284

briantang reviewed (changes requested):
The tests look good-to-go. The writerScenario1.spec failure where the test logs out is a strange intermittent one. I have created a separate task and assigned it to @csilva-ml .
https://project.marklogic.com/jira/browse/DHFPROD-8297
FYI @Sanjeevani19 . Needs a rebase in the meantime

briantang commented:
@lucianaabc-ml I am also seeing that the test is collapsing the "Buyer" panel so the next step cannot locate the property underneath. Can you try removing the line here that is clicking "[data-testid="Buyer-expand-icon"]" in **writerScenario1** ?

<img width="1288" alt="Screen Shot 2022-01-07 at 9 43 01 AM" src="https://user-images.githubusercontent.com/18374593/148584874-f31c2a09-c377-4d08-b435-0de1a2ccac6f.png">

Sanjeevani19 reviewed (commented):
@lucianaabc-ml UI tests are failing https://jenkins.marklogic.com/blue/organizations/jenkins/Datahub_CI/detail/PR-6400/23/tests, please look into it

lucianaabc-ml commented:
> @lucianaabc-ml UI tests are failing https://jenkins.marklogic.com/blue/organizations/jenkins/Datahub_CI/detail/PR-6400/23/tests, please look into it
I think @briantang was going to take a look into these, it seems that they are errors related to the React Testing Library, but it-s weird that they're failing cause I haven't done any other changes since the previous run and haven't done anything with the RTL  version.
